src/utils/resources.py

```diff
@@ -1214,6 +1214,15 @@ def __init__(self, properties: Dict[str, Any], *args, **kwargs):
             key: values for key, values in self.extras.items() if values["packages"]
         }
 
+        # Yarn repository is now provided by the core.
+        # Let's "move" any extra apt resources depending on yarn to the standard packages list.
+        for name, values in self.extras.items():
+            if values["repo"] == "deb https://dl.yarnpkg.com/debian/ stable main" \
+            and values["packages"] == [ "yarn" ]:
+                self.packages.append("yarn")
+                del self.extras[name]
```

Comment:
```
>>> D = {"foo": "foo", "bar": "bar"}
>>> for k, v in D.items():
...    print(k, v)
...    del D[k]
... 
foo foo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: dictionary changed size during iteration
```

:grimacing:

Reply:
Aaaaah we need to iterate over the keys instead…